doc/adr/0002-global-and-state-channel-snapshots.md

```diff
@@ -0,0 +1,37 @@
+# 2. Global and StateChannel snapshots
+
+Date: 2022-01-05
+
+## Status
+
+Proposed
+
+## Context
+
+Global snapshots and StateChannel snapshots should be defined with their
+responsibilities.
+
+## Decision
+
+### StateChannel snapshots
+
+L0 Cell can be identified as a proxy point between Layer 1 and Layer 0. Each
+state channel will have its own L0 Cell registered in the Layer 0. One of the
+responsibilities of the L0 Cell is to create a StateChannel snapshot. It can be
+performed either by proxying the snapshot received from Layer 1 or by
```

Comment:
If you mean that the state channel L1 is a linear blockchain then yes

Reply:
Yes, some kind of "majority" from layer 1.
I see that as an algorithm working on the L0 Cell that is receiving all StateChannel snapshots proposals from L1 (from all the state channel L1 peers), checks whether all required peers made the snapshot (checking against "next facilitators" written in snapshot n-1, and checks the majority. Then, L0 Cell return it as an output from the Cell and L0 layer can handle that. 
![image](https://user-images.githubusercontent.com/4959931/148601685-fa4d8379-951d-4d97-ab9a-b8b43fcbe171.png)
This is what I mean by "proxying" snapshot from L1 to L0. Basically from the L0 point of view, L0 Cell needs to propose a snapshot, no matter how it's obtained.